### PR TITLE
add v0.8 release notes

### PR DESCRIPTION
**Pull request by kevinintel**

### CODE REVIEW

File: release_notes/v0.8.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
+# OPEA Highlights
+- In this release, OPEA supports large vision models and provides an initial Agent microservice. It also integrates more components, such as vLLM on Ray, giving users more choices for different AI functionalities. We have expanded the application range, enabling ChatQnA on AIPC and validating it on AWS. Additionally, more examples, such as FAQGen, are coming. Deployment is important, Kubernetes manifests for ChatQnA, DocSum, CodeGen and CodeTrans are created.
```
Comment (irisdingbj) [inline release_notes/v0.8.md:2]: Can we add some highlights for GMC as well?  Like 
GenAI Microservices Connector (GMC) supports the launching, monitoring, and updating of GenAI microservice Applications on Kubernetes. It essentially supports a Kubernetes Custom Resource Definition for GenAI chains/pipelines that may be comprised of sequential, conditional, and parallel steps.

Comment (irisdingbj) [reply release_notes/v0.8.md:2]: @mkbhanda @yipan ^^

Comment (mkbhanda) [reply release_notes/v0.8.md]: Please add, based on @irisdingbj input 
"GenAI Microservices Connector (GMC) supports the launching, monitoring, and updating of GenAI microservice Applications on Kubernetes. It essentially supports a Kubernetes Custom Resource Definition for GenAI chains/pipelines that may be comprised of sequential, conditional, and parallel steps."

Comment (mkbhanda) [inline release_notes/v0.8.md]: Should we add that we tested OPEA pipeline works on Nvidia?

File: release_notes/v0.8.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+# GenAIExamples 
+- ChatQnA
+    - add ChatQnA instructions for AIPC([26d4ff](https://github.com/opea-project/GenAIExamples/commit/26d4ff11ffd323091d80efdd3f65e4c330b68840))
+    - Adapt Vllm response format ([034541](https://github.com/opea-project/GenAIExamples/commit/034541404e23ce3927c170237817e98f9323af26))
```
[the remark this answers is not in the document]
Comment (kevinintel) [reply release_notes/v0.8.md]: remove all commits

File: release_notes/v0.8.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# What’s New in OPEA v0.8
+- Broaden functionality coverage
+    - OPEA v0.8 Support LVM(Large Vision Models), it provides a microservice for llava on Intel Gaudi2. It also create an initial Agent microservice, OPEA now has the capability to process information, evaluate situations, and take actionable steps. Meanwhile more components are coming, such as vLLM on Ray, giving users more choices for different AI functionalities. FAQGen (Frequently Asked Questions Generation) is a new example of LLM application.
+- Deploy across more platforms
+    - We validate ChatQnA on AWS, including GenAI Microservices Connector (GMC), Kubernetes manifests and docker. Meanwhile it can work on AIPC. Kubernetes manifests for ChatQnA, DocSum, CodeGen and CodeTrans are created. The document for deploying OPEA with Docker Hub is also included in this release. GMC supports the launching, monitoring, and updating of GenAI microservice Applications on Kubernetes.
```
Comment (mkbhanda) [inline release_notes/v0.8.md]: s/validate/validated
s/AWS/AWS and Nvidia H100

Add one more sentence ..
This release also supports basic Authentication and Authorization using JWT tokens.

File: release_notes/v0.8.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kevinintel) [reply release_notes/v0.8.md]: remove coming

Comment (kevinintel) [reply release_notes/v0.8.md]: good idea

File: release_notes/v0.8.md (hunk 3 above)
Comment (kevinintel) [reply release_notes/v0.8.md]: not sure how to add AWS and Nvidia H100, current sentence is "OPEA pipeline can also operate on AIPC and Nvidia H100"
one more sentence has been added

File: release_notes/v0.8_details.md [hunk 4]
```diff
@@ -0,0 +1,192 @@
+# GenAIExamples
```
Comment (dbkinder) [inline release_notes/v0.8_details.md:1]: First (and only) H1 heading should be the document title, 
```suggestion
# OPEA v0.8 Release Notes Details

## GenAIExamples 
```

File: release_notes/v0.8_details.md [hunk 5]
```diff
@@ -0,0 +1,192 @@
...
+    - add docker compose yaml print for test([bb4230](https://github.com/opea-project/GenAIExamples/commit/bb42307af952a8ca8c80dec329d84e1fe94943f3))
+    - refactor translation ci test ([b7975e](https://github.com/opea-project/GenAIExamples/commit/b7975e79d8c75899961e5946d8ad0356065f20c5))
+    - refactor searchqna ci test([ecf333](https://github.com/opea-project/GenAIExamples/commit/ecf33388359a9bc20ff63676f169cc4d8129b1e7))
+    - Translate UT for UI([284d85](https://github.com/opea-project/GenAIExamples/commit/284d855bf410e5194c84523450397f0eb70ad0ee))
+    - enhancement the codetrans e2e test([450efc](https://github.com/opea-project/GenAIExamples/commit/450efcc139f26268b31a456db3f17024a37f896f))
+    - allow gmc e2e workflow to get secrets([f45f50](https://github.com/opea-project/GenAIExamples/commit/f45f508847823f3f6a1831d1a402932294b2a287))
+    - add checkout ref in gmc e2e workflow([62ae64](https://github.com/opea-project/GenAIExamples/commit/62ae64f13c8127cd7afd7d58d06c6cf9c51fafbf))
+    - SearchQnA UT([268d58](https://github.com/opea-project/GenAIExamples/commit/268d58d4a971d7d8340e72caf90a4fc14650612d))
+
+# GenAIComps
```
Comment (dbkinder) [inline release_notes/v0.8_details.md:60]:
```suggestion
## GenAIComps 
```

File: release_notes/v0.8_details.md [hunk 6]
```diff
@@ -0,0 +1,192 @@
...
+    - align asr output and llm input without using orchestrator([64e042](https://github.com/opea-project/GenAIComps/commit/64e042146f4a7ea40e70a7fc5431d7f32e8ee02c))
+    - doc: add missing in README.md codeblock([2792e2](https://github.com/opea-project/GenAIComps/commit/2792e28334760d94908aa521be1bedcec8848ad3))
+
+- CI/UT
+    - fix duplicate ci test([33f37c](https://github.com/opea-project/GenAIComps/commit/33f37cebd4bba515b21203f94af2616faade2baa))
+    - build and push new docker images into registry([80da5a](https://github.com/opea-project/GenAIComps/commit/80da5a86abafeceaf196bacc17e3922dd3173be8))
+    - update image build for gaudi([fe3d22](https://github.com/opea-project/GenAIComps/commit/fe3d22acabdee2fbf72ced0fae3832e7ca1fa3e4))
+    - add guardrails ut([556030](https://github.com/opea-project/GenAIComps/commit/55603000eba4823678b3e79623186fa591a2f06f))
+
+# GenAIEvals
```
Comment (dbkinder) [inline release_notes/v0.8_details.md:109]:
```suggestion
## GenAIEvals 
```

File: release_notes/v0.8_details.md [hunk 7]
```diff
@@ -0,0 +1,192 @@
...
+    - build and push new docker images into registry([80da5a](https://github.com/opea-project/GenAIComps/commit/80da5a86abafeceaf196bacc17e3922dd3173be8))
+    - update image build for gaudi([fe3d22](https://github.com/opea-project/GenAIComps/commit/fe3d22acabdee2fbf72ced0fae3832e7ca1fa3e4))
+    - add guardrails ut([556030](https://github.com/opea-project/GenAIComps/commit/55603000eba4823678b3e79623186fa591a2f06f))
+
+# GenAIEvals 
+- Update lm-eval to 0.4.3([89c825](https://github.com/opea-project/GenAIEval/commit/89c8255f3f41a545ace25c61db3160cbece3047f))
+- add toxicity/bias/hallucination metrics([48015a](https://github.com/opea-project/GenAIEval/commit/48015a1cb0c200aa1e7929367acd68d971ae544c))
+- Support stress benchmark test([59cb27](https://github.com/opea-project/GenAIEval/commit/59cb275ca870bc1ff4514a1e3b8c67ca9e48c71e)) 
+
+# GenAIInfra
```
Comment (dbkinder) [inline release_notes/v0.8_details.md:114]:
```suggestion
## GenAIInfra 
```

**Review by dbkinder — CHANGES_REQUESTED** [review]
Only one H1 heading, (for the title), the others should be H2

File: release_notes/v0.8.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
+# What’s New in OPEA v0.8
+
+- Broaden functionality
+    - Supported frequently asked questions (FAQs) generation GenAI example
+    - Expanded the support of LLMs such as Llama3.1 and Qwen2 and supported LVMs such as llava
+    - Enabled end-to-end peformance and accuracy benchmarking
+    - Supported the experimental Agent microservice and GenAI example
```
Comment (dbkinder) [inline release_notes/v0.8.md:7]:
```suggestion
    - Supports the experimental Agent microservice and GenAI example
```

File: release_notes/v0.8.md [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+    - Supported LLM serving on Ray
+
+- Multi-platform support
+    - Released the docker images of GenAI components under OPEA dockerhub and supported the deployment with docker
```
Comment (dbkinder) [inline release_notes/v0.8.md:11]: Also, can you add a link to the OPEA dockerhub here:
```suggestion
    - Releases the Docker images of GenAI components under OPEA dockerhub and supports the deployment with Docker
```

File: release_notes/v0.8.md [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+    - Supported the cloud native deployment through Kubernetes manifests and GenAI Microservices Connector (GMC)
```
Comment (dbkinder) [inline release_notes/v0.8.md:12]:
```suggestion
    - Supports cloud-native deployment through Kubernetes manifests and GenAI Microservices Connector (GMC)
```

File: release_notes/v0.8.md [hunk 11]
```diff
@@ -0,0 +1,16 @@
...
+    - Enabled the experimental authentication and authorization support using JWT tokens
```
Comment (dbkinder) [inline release_notes/v0.8.md:13]:
```suggestion
    - Enables the experimental authentication and authorization support using JWT tokens
```

File: release_notes/v0.8.md [hunk 12]
```diff
@@ -0,0 +1,16 @@
...
+    - Validated ChatQnA on multiple platforms such as Xeon, Gaudi, AIPC and Nvidia, and AWS
```
Comment (dbkinder) [inline release_notes/v0.8.md:14]:
```suggestion
    - Validates ChatQnA on multiple platforms such as Xeon, Gaudi, AIPC, Nvidia, and AWS
```

**Review by dbkinder — DISMISSED** [review]
Do the details need to be in a separate document?  It seems better to just put them at the end of the other document and give it the title:  "# OPEA Release Notes v0.8" and add an H2 heading of "## What's New"

Also, it's better to use the present tense for these bullet items (not the past)_

File: release_notes/v0.8.md (hunk 12 above)
Comment (kevinintel) [reply release_notes/v0.8.md:14]: It's title of PR, should we change it?

File: release_notes/v0.8.md (hunk 11 above)
Comment (kevinintel) [reply release_notes/v0.8.md:13]: It's title of PR, should we change it?

File: release_notes/v0.8.md (hunk 10 above)
Comment (kevinintel) [reply release_notes/v0.8.md:12]: It's title of PR, should we change it?

File: release_notes/v0.8.md (hunk 9 above)
Comment (kevinintel) [reply release_notes/v0.8.md:11]: It's title of PR, should we change it?

File: release_notes/v0.8.md [hunk 13]
```diff
@@ -0,0 +1,16 @@
+# What’s New in OPEA v0.8
+
+- Broaden functionality
+    - Supported frequently asked questions (FAQs) generation GenAI example
+    - Expanded the support of LLMs such as Llama3.1 and Qwen2 and supported LVMs such as llava
+    - Enabled end-to-end peformance and accuracy benchmarking
+    - Supported the experimental Agent microservice and GenAI example
+    - Supported LLM serving on Ray
```
[the remark this answers is not in the document]
Comment (kevinintel) [reply release_notes/v0.8.md:8]: It's title of PR, should we change it?

File: release_notes/v0.8.md (hunk 8 above)
Comment (kevinintel) [reply release_notes/v0.8.md:7]: It's title of PR, should we change it?

File: release_notes/v0.8.md [hunk 14]
```diff
@@ -0,0 +1,16 @@
+# What’s New in OPEA v0.8
+
+- Broaden functionality
+    - Supported frequently asked questions (FAQs) generation GenAI example
+    - Expanded the support of LLMs such as Llama3.1 and Qwen2 and supported LVMs such as llava
+    - Enabled end-to-end peformance and accuracy benchmarking
```
[the remark this answers is not in the document]
Comment (kevinintel) [reply release_notes/v0.8.md:6]: It's title of PR, should we change it?

File: release_notes/v0.8.md [hunk 15]
```diff
@@ -0,0 +1,16 @@
+# What’s New in OPEA v0.8
+
+- Broaden functionality
+    - Supported frequently asked questions (FAQs) generation GenAI example
+    - Expanded the support of LLMs such as Llama3.1 and Qwen2 and supported LVMs such as llava
```
[the remark this answers is not in the document]
Comment (kevinintel) [reply release_notes/v0.8.md:5]: It's title of PR, should we change it?

File: release_notes/v0.8.md [hunk 16]
```diff
@@ -0,0 +1,16 @@
+# What’s New in OPEA v0.8
+
+- Broaden functionality
+    - Supported frequently asked questions (FAQs) generation GenAI example
```
[the remark this answers is not in the document]
Comment (kevinintel) [reply release_notes/v0.8.md:4]: It's title of PR, should we change it?

File: release_notes/v0.8.md (hunk 14 above)
[the remark this answers is not in the document]
Comment (hshen14) [reply release_notes/v0.8.md:6]: Looks good to me.

**Comment by kevinintel** [comment]
> Do the details need to be in a separate document? It seems better to just put them at the end of the other document and give it the title: "# OPEA Release Notes v0.8" and add an H2 heading of "## What's New"
> 
> Also, it's better to use the present tense for these bullet items (not the past)_

I putted it in the same document 2 days ago， but someone complained the long list. That‘s why moving to another doc.

**Comment by dbkinder** [comment]
> > Do the details need to be in a separate document? It seems better to just put them at the end of the other document and give it the title: "# OPEA Release Notes v0.8" and add an H2 heading of "## What's New"
> > Also, it's better to use the present tense for these bullet items (not the past)_
> 
> I putted it in the same document 2 days ago， but someone complained the long list. That‘s why moving to another doc.

It's still a long list :)  I'd vote to keep this together in one release notes document.  People can stop reading if they're not interested in the details.

**Comment by kevinintel** [comment]
> > > Do the details need to be in a separate document? It seems better to just put them at the end of the other document and give it the title: "# OPEA Release Notes v0.8" and add an H2 heading of "## What's New"
> > > Also, it's better to use the present tense for these bullet items (not the past)_
> > 
> > 
> > I putted it in the same document 2 days ago， but someone complained the long list. That‘s why moving to another doc.
> 
> It's still a long list :) I'd vote to keep this together in one release notes document. People can stop reading if they're not interested in the details.

done

**Review by hshen14 — APPROVED** [review]

File: release_notes/v0.8.md [hunk 17]
```diff
@@ -0,0 +1,303 @@
...
+    - Support multiple router service in one namespace([0ac732](https://github.com/opea-project/GenAIInfra/commit/0ac73213b501fb5949a5ac8bf7f52d5a4acef580)) 
+    - Improve workflow trigger conditions to be more precise([ab5c8d](https://github.com/opea-project/GenAIInfra/commit/ab5c8d8c07d8f8353315b7ebaf1eb745bf7b28e5))
+    - Remove unnecessary component DocSumGaudi which would cause error([9b973a](https://github.com/opea-project/GenAIInfra/commit/9b973aceb25c307f2c7692c9364ebac9040b9a5d))
+    - Remove chart_test scripts and add script to dump pod status([88caf0](https://github.com/opea-project/GenAIInfra/commit/88caf0df947866ffe609cf60805282970f887429))
+</details>
+
+## Thanks to these contributors
+This release includes valuable contributions from the Intel core team.. We would also like to thank everyone who reported issues or provided feedback. 
+
+Contributors
```
Comment (dbkinder) [inline release_notes/v0.8.md:231]: Is this a list of contributors across all the repos? I'm not sure we need this full list of 70 contributors, maybe just the top 10 (if any)?

Comment (hshen14) [reply release_notes/v0.8.md]: I think it's ok to put the contributors here with the small icons.

**Review by hshen14 — APPROVED** [review]

**Review by chensuyue — APPROVED** [review]

**Review by ftian1 — APPROVED** [review]

**Pull request merged**